Resources/Prototypes/_NF/Shipyard/Sr/bottleneck.yml

```
@@ -0,0 +1,23 @@
+- type: vessel
+  id: Bottleneck
+  name: NT Bottleneck
+  description: A personal transport and mobile office for the station representative
```

Capitalize proper nouns.
```suggestion
  description: A personal transport and mobile office for the Station Representative.
```

---

There's a lot of inconsistency when it comes to ship descriptions. If you delve in, it's quite terrifying. Lots of them go without full stops at the end of their descriptions. I'll commit this one.